composer.json

```diff
@@ -23,9 +23,12 @@
     ],
     "require": {
         "php": "^7.2",
-        "doctrine/common": "^2.7",
+        "doctrine/annotations": "^1.10",
```

Comment:
Are we using classes from this namespace?

Reply:
I didn't see any usage on the code

Reply:
You're right, I'll remove this.